src/cli.py

```diff
@@ -7,14 +7,24 @@
 
 import click
 
-from constants import SEPARATOR
+
+def hide_header(ctx, param, value):
```

Comment:
I would name it instead `show_header`.

In my mind, the default is show the header (since default format is `pretty`).
Optionally we hide it.